src/app/view/components/_common/iframe/iframe-porlet-legacy/iframe-porlet-legacy.component.ts

```diff
@@ -106,30 +106,23 @@ export class IframePortletLegacyComponent implements OnInit {
             .withLatestFrom(this.route.parent.url.map((urlSegment: UrlSegment[]) => urlSegment[0].path))
             .flatMap(([id, url]) => {
                 this.dotLoadingIndicatorService.show();
-                return url === 'add'
-                    ? this.contentletService.getUrlById(id)
-                    : this.dotNavigationService.getUrlById(id);
+                return url === 'add' ? this.contentletService.getUrlById(id) : this.dotNavigationService.getUrlById(id);
             })
             .subscribe((url: string) => {
                 this.setUrl(url);
             });
     }
+
+    /**
```

Comment:
Private methods shouldn't have doc

Reply:
In general private method does not need to be documented, however documentation is good!

Reply:
I prefer to keep a comment on this one, to remember why we are removing the iframe from the DOM with the ngIf.